dev/parm/config/gfs/config.efcs

```diff
@@ -43,8 +43,8 @@ if [[ ${RUN} == "enkfgfs" ]]; then
 fi
 
 # Use serial I/O for ensemble (lustre?)
-export OUTPUT_FILETYPE_ATM="netcdf"
-export OUTPUT_FILETYPE_SFC="netcdf"
+#export OUTPUT_FILETYPE_ATM="netcdf"
+#export OUTPUT_FILETYPE_SFC="netcdf"
```

Comment:
I think I'm OK with removing this.  I believe this was added when we were using netCDF v4.7.4, which had a tendency to hang on parallel writes on Lustre.  Has this been tested at high resolution on one of the Lustre systems (Ursa, Hera, or MSU) at high resolution?

Reply:
I think it's good to keep it commented out but present.  We might end up needing it for high res Ursa testing if/when we get there.  Also different netcdf modules in the future might behave differently.

Reply:
Thanks @CatherineThomas-NOAA, I agree.